src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Models/KeyValueSelector.cs

```diff
@@ -24,6 +24,11 @@ public class KeyValueSelector
         /// </summary>
         public string SnapshotName { get; set; }
 
+        /// <summary>
+        /// A boolean that signifies whether this selector is intended to select feature flags.
+        /// </summary>
+        public bool IsFeatureFlagSelector { get; set; }
```

Comment:
This looks like a workaround to avoid having to have two lists of key value selectors.

Reply:
I guess so, but I think there's either an extra property needed or 3 lists of selectors. We need to get kv selectors only, ff selectors only, or both of them in order of how the user specified them to match the existing behavior of loading from app config.

Reply:
Discussed offline: The issue here is that since the beginning we have merged key-value and feature flag selectors into a single list that is executed in order by the configuration provider. By separating out in two lists we lose ordering and therefore introduce a breaking behavioral change regarding setting and feature flag precedence.